packages/client/modules/meeting/components/PokerTemplateItem.tsx

```diff
@@ -94,6 +95,14 @@ const PokerTemplateItem = (props: Props) => {
       store.get(teamId)?.setValue(null, 'editingScaleId')
     })
   }
+  useEffect(() => {
+    if (!isActive) return
+    SendClientSegmentEventMutation(atmosphere, 'Viewed Template', {
+      meetingType: 'poker',
+      scope,
+      templateName
+    })
```

Comment:
+1 I'd love to add `isFree` to the parameters we're sending to Segment.

Reply:
Sure thing - added!